website/project/model.py

```diff
@@ -1964,7 +1964,7 @@ def get_recent_logs(self, n=10):
 
         :param int n: Number of logs to retrieve
         """
-        return list(reversed(self.logs)[:n])
+        return list(reversed(self.logs))[:n]
```

Comment:
`reversed` should be used here either, ODM doesn't support it AFAIK.
`self.logs.sort('-date')[:n]` should do the trick here